addons/crm_livechat/static/tests/message_tests.js

```diff
@@ -0,0 +1,30 @@
+/* @odoo-module */
+
+import { startServer } from "@bus/../tests/helpers/mock_python_environment";
+import { start } from "@mail/../tests/helpers/test_utils";
+import { click, contains, insertText } from "@web/../tests/utils";
+
+QUnit.module("message");
+
+QUnit.test("Can open lead from internal link", async function (assert) {
```

Comment:
If needed: I can adapt the test `Can execute lead command` in `addons/crm_livechat/static/tests/composer_tests.js` instead of creating another test.

Reply:
It's fine to have this test, and it looks much cleaner now with the last updates!